### PR TITLE
CP-110 Add API documentation

### PR DESCRIPTION
**Pull request by RobLoach**
- [JIRA CP-110](https://kalamuna.atlassian.net/browse/CP-110)
- [Preview](https://github.com/pantheon-systems/pantheon-edge-integrations/blob/6a1afa642429381434f422f25f6cfdddf1df92a8/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,21 +2,96 @@
...
 ### returnVaryHeader($key)
+
 Returns vary header array, based on header data.
 
+### Global Methods
+
+There are also static methods defined within the class to help assist in retrieving data without having to instantiate the object yourself.
+
+``` php
+HeaderData::personalizationObject()
```
Comment (jazzsequence) [inline README.md]: I'd suggest spelling these out a little bit more. The naming convention is such that it's pretty obvious which method each of these is a wrapper for, but I would still add a brief description of each just to make that 100% clear.

**Review by jazzsequence — COMMENTED** [review]
Docs look great! Just one minor suggestion/addition.

File: README.md (hunk 1 above)
Comment (RobLoach) [reply README.md]: Agreed. Exploded them a bit.

**Review by jazzsequence — APPROVED** [review]
:shipit: 🚀

**Pull request merged**